playbook.yml

```diff
@@ -19,7 +19,12 @@
       debug: "True"
       redis_host: "{{ meta.name }}-redis"
       redis_port: 6379
-      redis_password: ''
+      redis_password: ""
+      TOKEN_SIGNATURE_ALGORITHM: "ES256"
+      PUBLIC_KEY_PATH: "/etc/pulp/keys/container_auth_public_key.pem"
+      PRIVATE_KEY_PATH: "/etc/pulp/keys/container_auth_private_key.pem"
```

Comment:
are we generating these keys?

Reply:
The operator is not generating these keys. The user must provide a secret with the private/public keys. They will then be mounted in this location.

Mentioned in the example above:
```
kubectl create secret generic container-auth \\n--from-file=container_auth_private_key.pem=./container_auth_private_key.pem \\n--from-file=container_auth_public_key.pem=./container_auth_public_key.pem
```

Reply:
Oh! sorry! I misread the example